build.sbt

```diff
@@ -87,7 +87,7 @@ lazy val root = Project("pan-domain-auth-root", file(".")).aggregate(
   exampleApp
 ).settings(
   publish / skip := true,
-  // releaseVersion := ReleaseVersion.fromAggregatedAssessedCompatibilityWithLatestRelease().value,
+  releaseVersion := ReleaseVersion.fromAggregatedAssessedCompatibilityWithLatestRelease().value,
```

Comment:
Re-enabling this after we had to disable it for https://github.com/guardian/pan-domain-authentication/pull/165 - the new Scala 3 artifacts are available:

* https://github.com/guardian/gha-scala-library-release-workflow/issues/33